.storybook/preview.ts

```diff
@@ -0,0 +1,20 @@
+import type { Preview } from "@storybook/react-vite";
+// (global.css) import 예정
+
+const preview: Preview = {
+  parameters: {
```

Comment:
```suggestion
  parameters: {
  actions: { argTypesRegex: '^on[A-Z].*' },
}
```
이 설정은 storybook에서 onClick, onChange 같은 이벤트 핸들러를 자동으로 Action 패널에 매핑해주는 옵션입니다!!!
최신 버전의 storybook에서는 대부분의 요런 핸들러를 자동으로 감지하지만, 
컴포넌트 구조나 래핑 방식에 따라 감지가 실패하는 경우가 있어서 미연에 이런 문제를 방지하려고 기본값으로 추가해두는 경우가 많은 것 같아요!!

우리 프로젝트도 이 설정을 넣어두면 이벤트 감지가 더 안정적으로 동작할 것 같은데 어떻게 생각하시나욤?!! 🧐

Reply:
@mimizae 리뷰하면서 suggestion 기능은 처음 보는게 이거 무지 편하네요...!! 짱이다

Reply:
`argTypesRegex` 옵션을 넣는 편이 확실히 더 안정적일 것 같습니당 반영해두도록 할게요 !! 

p.s. 저도 코드 제안할 때마다 마크다운 코드 문법으로만 작성했는데 호오... 너무 좋네요 (^///^)

Reply:
와!!!!! 오엠지 야무진 설명!!!!!!!!!!!! 버전까지 신경 쓰진 못했네요... 다음부터는 좀 더 세밀히 리뷰해보겠습니다 🥰 어푸어푸